MIP113/MIP113.md

```diff
@@ -312,11 +312,11 @@ AVC splits are disabled until the launch of NewChain.
 
 #### 5.2.1
 
-To become recognized as an Unaffiliated AVC Member, an Ecosystem Actor must post a recognition submission message publicly on the Maker Governance Forum.
+To become recognized as an Unaffiliated AVC Member or to join an existing AVC, an Alignment Conserver must post a recognition submission message publicly on the Maker Governance Forum.
 
 ##### 5.2.1.1
 
-A recognition submission message must be cryptographically signed by an Ethereum address containing at least 1 MKR, or that has delegated at least 1 MKR.
+A recognition submission message must be cryptographically signed by an Ethereum address containing at least 1 MKR, or that has delegated at least 1 MKR. This condition must be met during all mandate statuses, including 'active' and 'pending'.
```

Comment:
This change is not relevant at the point someone applies to become a member of an AVC. These statuses apply to the AVC, not the members.

Reply:
Thanks for the reply. I wasn't clear enough, what I tried to specify is that AVC members should met the condition despite the status of their AVC being "active" or "pending".